File: 5kuy/Human_Readable_Time/test_solution.py
import solution



def test_solution():
    assert(solution.make_readable(0)      == "00:00:00")
    assert(solution.make_readable(59)     == "00:00:59")
    assert(solution.make_readable(60)     == "00:01:00")
    assert(solution.make_readable(3599)   == "00:59:59")
    assert(solution.make_readable(3600)   == "01:00:00")
    assert(solution.make_readable(86399)  == "23:59:59")
    assert(solution.make_readable(86400)  == "24:00:00")
    assert(solution.make_readable(359999) == "99:59:59")
    print("All test passed!")


if __name__ == '__main__':
    test_solution()

File: 6kuy/Persintent_Bugger/test_solution.c
#include <stdio.h>
#include <assert.h>

#include "solution.c"

int main(int argc, char *argv[]){
    assert(persistence(39) == 3);
    assert(persistence(4) == 0);
    assert(persistence(25) == 2);
    assert(persistence(999) == 4);
    printf("All test passed\n");
    return 0;
}
